src/core.rs

```diff
@@ -114,11 +114,16 @@ impl<T> Webauthn<T> {
     pub fn generate_challenge_register(
         &self,
         user_name: &str,
-        policy: Option<UserVerificationPolicy>,
+        user_verification_required: bool,
```

Comment:
any reason we  don't  just pass the policy directly here?

Reply:
This is the "basic" register function, so I thought to keep it as UV=true/false for discouraged/required to make it simpler to hold. The policy can be directly used in the options version.